pkg/ccl/backupccl/restore_job.go

```diff
@@ -400,6 +400,15 @@ func restore(
 			}
 
 			mu.res.Add(progDetails.Summary)
+
+			// After the change to bound the memory used by SST iterator, we can now
+			// get a progress entry for a partially completed span. Do not mark the
+			// span as done when receiving these entries.
+			if !progDetails.Done {
```

Comment:
should sync up with @msbutler -- we were just talking about how this should maybe the a timestamp and then done := timestamp == endtime

Reply:
resolved and now sending the timestamp